src/board/system76/lemp9/keymap/levi.c

```diff
@@ -0,0 +1,24 @@
+// Default layout
+
+#include <board/keymap.h>
+
+uint16_t __code KEYMAP[KM_LAY][KM_OUT][KM_IN] = {
+LAYOUT(
+    K_ESC, K_F1, K_F2, K_F3, K_F4, K_F5, K_F6, K_F7, K_F8, K_F9, K_F10, K_F11, K_F12, K_HOME, K_END, K_PRINT_SCREEN, K_DEL,
+    K_TICK, K_1, K_2, K_3, K_4, K_5, K_6, K_7, K_8, K_9, K_0, K_BRACE_OPEN, K_BRACE_CLOSE, K_BKSP,
+    K_TAB, K_QUOTE, K_COMMA, K_PERIOD, K_P, K_Y, K_F, K_G, K_C, K_R, K_L, K_SLASH, K_EQUALS, K_BACKSLASH,
+    K_LEFT_CTRL, K_A, K_O, K_E, K_U, K_I, K_D, K_H, K_T, K_N, K_S, K_MINUS, K_ENTER,
+    K_LEFT_SHIFT, K_SEMICOLON, K_Q, K_J, K_K, K_X, K_B, K_M, K_W, K_V, K_Z, K_RIGHT_SHIFT,
+    KT_FN, K_Z, K_LEFT_ALT, K_LEFT_SUPER, K_SPACE, K_RIGHT_ALT, K_RIGHT_CTRL, K_PGUP, K_UP, K_PGDN,
```

Comment:
This extra Z is up to something

Reply:
It was intentional, but it is definitely strange. It allows me to `Ctrl` + `Z` with my left hand, since Dvorak migrated `Z` all the way over to where `/` is on QWERTY. My QMK layouts often have something similar: https://github.com/leviport/qmk_firmware/blob/bc5ee826801b703340c9f50a0dec2ed0e82c9fb8/keyboards/ergodox_ez/keymaps/levi_layout/keymap.c#L42